docs/docusaurus.config.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
         docs: {
           routeBasePath: "/",
           sidebarPath: require.resolve("./sidebars.js"),
-          editUrl: "https://github.com/awslabs/go-research-on-aws/website/",
+          editUrl: "https://github.com/awslabs/service-workbench-on-aws-cn/edit/develop/docs/",
```

Comment:
Can we use relative path like `/edit/...`

Reply:
I am not sure I fully understand your point.
The online docs link prefix is not the same with github source code.
Doc link prefix is : "https://awslabs.github.io/service-workbench-on-aws-cn"
Source edit link prefix is:  "https://github.com/awslabs/service-workbench-on-aws-cn",

So if we use **editUrl: "/service-workbench-on-aws-cn//edit/develop/docs/"**, edit link will be "https://awslabs.github.io/service-workbench-on-aws-cn/edit/...". This link is not correct for editing source code.

Reply:
Hardcore the repo url always is bad. Looks like it's some limit of docusaurus. Let's move to latest version to see what happens.

Ideally it only need a global repo make it work, like how mkdocs does.

https://github.com/awslabs/quantum-computing-exploration-for-drug-discovery-on-aws/blob/6a37461b8bd6382ece040035c6accd4e9de67649/docs/mkdocs.base.yml#L3